rhel8/profiles/cis.profile

```diff
@@ -804,7 +804,7 @@ selections:
 
 
     ### 5.2.19 Ensure SSH MaxSessions is set to 4 or less (Scored)
-
+    - sshd_set_max_sessions=4
```

Comment:
Sorry for not being clear here.
The profile needs to select the rule and refine the Value, so:
```
     - sshd_set_max_sessions
     - var_sshd_set_max_sessions=4
```

Reply:
Resolved in 0e95def